internal/resource/config/serversettings/server_settings_resource.go

```diff
@@ -860,10 +797,7 @@ func (r *serverSettingsResource) ValidateConfig(ctx context.Context, req resourc
 	// Validate contact_info email value
 	ciEmail := model.ContactInfo.Attributes()["email"]
 	if internaltypes.IsDefined(ciEmail) {
-		ciEmailStringValue := ciEmail.(types.String)
-		if internaltypes.IsNonEmptyString(ciEmailStringValue) && !internaltypes.IsEmailFormat(ciEmailStringValue.ValueString()) {
-			resp.Diagnostics.AddError("Invalid Email Format!", fmt.Sprintf("Please provide a valid email address - \"%s\" needs to be in a valid email format according to RFC 5322.  For example, \"<user>@<company>.<tld>\"", ciEmailStringValue.ValueString()))
-		}
+		validators.IsEmailFormat(ciEmail, resp)
```

Comment:
Do we need these validation calls when we already have validators in the schema?